pkg/event/target/amqp.go

```diff
@@ -42,6 +45,8 @@ type AMQPArgs struct {
 	Internal     bool     `json:"internal"`
 	NoWait       bool     `json:"noWait"`
 	AutoDeleted  bool     `json:"autoDeleted"`
+	QueueDir     string   `json:"queueDir"`
+	QueueLimit   uint16   `json:"queueLimit"`
```

Comment:
is QueueLimit 65535? Why?

Reply:
The maximum value that QueueLimit can have is 10000 which is the max limit for the store. Do you think we should be more specific?

Reply:
Is it hard limit i.e. when > 10000 is passed to `QueueLimit`, is it an error? If yes, are you handling it?

What is the meaning of 0 to `QueueLimit`? disabled or use default? Have you described these anywhere as a comment?

Reply:
1 - No, we are not validating this. I will add such validation in args.Validate func to avoid such setting
2 - 0 will take the default value - 10000, I have explained this in docs.